osbs/core.py

```diff
@@ -494,7 +494,12 @@ def logs(self, build_id, follow=False, build_json=None, wait_if_missing=False):
         buildlogs_url = self._build_url("builds/%s/log/" % build_id)
         response = self._get(buildlogs_url, headers={'Connection': 'close'})
         check_response(response)
-        return response.content
+
+        content = response.content
+        if isinstance(content, bytes):
+            return content
+        else:
+            return content.encode('utf-8')
```

Comment:
Why do we need to encode it this early? We shouldn't encode it until we need to serialize it.